tests/test_result_to_dict.py

```diff
@@ -154,6 +162,16 @@ def test_base_db_entity(self):
             self.assertEqual(converted_data[0]['data_bool'], data_bool)
             self.assertEqual(converted_data[0]['data_unicode'], data_unicode)
 
+    def test_callback(self):
+        json_value = {"userId": 1, "id": 1, "title": "Some Title", "object": '{"value": "JSON in Python", '
+                                                                             '"number": 123}'}
+        data = result_to_dict(json_value, callback=get_object)
+        self.assertNotEqual(data, None)
+        self.assertIsInstance(data, dict)
+        self.assertDictEqual(data, {"value": "JSON in Python", "number": 123})

```

Comment:
question. so here `data` is `json_value.object`?

Reply:
yes, the string is converted to object